src/pages/index.js

```diff
@@ -41,62 +77,205 @@ const userInformation = new UserInfo(
   constants.profileDescription
 );
 
-//EDIT PROFILE POPUPWITHFORM.JS
+//AVATAR INFO
 
-const editProfile = new PopupWithForm(constants.profileEditModal, (data) => {
-  userInformation.setUserInfo(data);
-  editProfile.close();
-});
+const avatarInformation = new UserInfo("","",constants.avatarImg);
```

Comment:
`const avatarInformation = new UserInfo("","",constants.avatarImg);
`

`avatarInformation` is not needed because you have already created an instance of `UserInfo`. Please, pass the 3rd selector iin line 77